assets/ui/etjump_settings_5.menu

```diff
@@ -81,6 +85,33 @@ menuDef {
         MULTI               (SUBW_ITEM_RIGHT_X, SUBW_STRAFEQUALITY_ITEM_Y + (SUBW_ITEM_SPACING_Y * 5), SUBW_ITEM_WIDTH, SUBW_ITEM_HEIGHT, "Strafe quality size:", 0.2, SUBW_ITEM_HEIGHT, "etj_strafeQualitySize", cvarFloatList { "Tiny" 1 "Small" 2 "Medium" 3 "Big" 4 }, "Sets size of strafe quality\netj_strafeQualitySize")
         MULTI               (SUBW_ITEM_RIGHT_X, SUBW_STRAFEQUALITY_ITEM_Y + (SUBW_ITEM_SPACING_Y * 6), SUBW_ITEM_WIDTH, SUBW_ITEM_HEIGHT, "Strafe quality style:", 0.2, SUBW_ITEM_HEIGHT, "etj_strafeQualityStyle", cvarFloatList { "Full" 0 "Percent" 1 "Number" 2 }, "Strafe quality style\netj_strafeQualityStyle")
 
+    SUBWINDOW(SUBW_RECT_RIGHT_X, SUBW_UPMOVEMETER_Y, SUBW_WIDTH, SUBW_UPMOVEMETER_HEIGHT, "UPMOVE METER")
+        MULTI               (SUBW_ITEM_RIGHT_X, SUBW_UPMOVEMETER_ITEM_Y + (SUBW_ITEM_SPACING_Y * 0), SUBW_ITEM_WIDTH, SUBW_ITEM_HEIGHT, "Upmove meter:", 0.2, SUBW_ITEM_HEIGHT, "etj_drawUpmoveMeter", cvarFloatList { "Off" 0 "Bar" 1 "Text" 2 "Bar/Text" 3 }, "Draw upmove meter\netj_drawUpmoveMeter")
```

Comment:
Small nitpick: other bitflag cvars in menus use `flag + flag` format rather than `flag/flag`